core/src/main/java/apoc/export/graphml/XmlGraphMLReader.java

```diff
@@ -337,6 +337,7 @@ public long parseXML(Reader input, TerminationGuard terminationGuard) throws XML
             throw e;
         } finally {
             tx.close();
+            reader.close();
```

Comment:
If `tx.close()` fails (and I believe it can under certain circumstances) we won't close the reader now. You can avoid this by using try with resources (`try (final var reader = ...) {`), or using something like `IOUtils.closeAll` (from mono repo).

Reply:
This reader isn't auto closeable which is why I didn't wrap it in the try, it is just freeing up resources (not closing the file, which I made a part of a try with resources already), maybe it is fine?

Reply:
Sounds fine :).